i18n/fr/docusaurus-plugin-content-docs/version-21.10/releases/centreon-os-extensions.md

```diff
@@ -38,11 +38,12 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
 - [Install] Added ability to use a non default database name
 - [Widget] Added an alarm duration filter in widget configuration
 - [Widget] Added an option to schedule a check when you open a ticket or acknowledge a resource
-- [Widget] Preselect option if only one choice is available
+- [Widget] Added preselect option in select box if only one choice is available
 
 #### Bug fixes
 
 - [Provider] Fixed error 400 issue when opening a ticket using Jira provider
+- [Provider] Fixed regression with Mail provider
 - [Widget] Display of the state type of the host instead of the service
```

Comment:
Should it be "Fixed the state type that displayed the host instead of the service"?

Reply:
no, previously the column state display service value instead of host value.